server/load_api_data_to_db.py

```diff
@@ -1,22 +1,51 @@
 import json
 import sqlite3
+from typing import Any, Union
 
-con: sqlite3.Connection = sqlite3.connect("data/api_data.db")
-con.execute(
-    """
-CREATE TABLE IF NOT EXISTS api_data (
-    key     VARCHAR(30) UNIQUE PRIMARY KEY NOT NULL,
-    value   BLOB NOT NULL
-);""",
-)
-# we are using this file in docker, so we don't want to use an acid compliant database ;)
-con.execute("""PRAGMA journal_mode = OFF;""")
-con.execute("""PRAGMA synchronous = OFF;""")
-
-with open("data/api_data.json", encoding="utf-8") as file:
-    data = json.load(file)
-    new_data: list[tuple[str, str]] = [(key, json.dumps(value)) for key, value in data.items()]
-
-with con:
-    con.executemany("INSERT INTO api_data(key, value) VALUES (?,?)", new_data)
-print("Initialized KV store")
+
+def add_to_database(new_data):
+    """add data consisting of (key, de, en) to the sqlite database"""
+    con: sqlite3.Connection = sqlite3.connect("data/api_data.db")
+    con.execute(
+        """
+    CREATE TABLE IF NOT EXISTS api_data (
+        key     VARCHAR(30) UNIQUE PRIMARY KEY NOT NULL,
+        de      BLOB NOT NULL,
+        en      BLOB NOT NULL
+    );""",
+    )
+    # we are using this file in docker, so we don't want to use an acid compliant database ;)
+    con.execute("""PRAGMA journal_mode = OFF;""")
+    con.execute("""PRAGMA synchronous = OFF;""")
+
+    with con:
+        con.executemany("INSERT INTO api_data(key, de, en) VALUES (?,?,?)", new_data)
+
+
+def localise(value: Union[str, list[Any], dict[str, Any]], language: str) -> Any:
+    """Recursively localise a dictionary"""
+    if isinstance(value, (bool, float, int, str)) or value is None:
+        return value
+    if isinstance(value, list):
+        return [localise(v, language) for v in value]
+    if isinstance(value, dict):
+        # since we only localise strings, the default to the empty string is safe
+        if "en" in value or "de" in value:
+            return value.get(language, "")
```

Comment:
We said in a previous PR, that sometimes it could make sense to only have a comment, for example for one language.
I admit that this is faulty, as it returns an empty string for these cases, but I have not found a way to implement this that is not terribly inelegant. I am up for suggestions..